python/gui/auto_generated/layout/qgslayoutdesignerinterface.sip.in

```diff
@@ -109,6 +109,13 @@ Returns whether the atlas preview mode is enabled in the designer.
 .. seealso:: :py:func:`setAtlasPreviewEnabled`
 
 .. versionadded:: 3.4
+%End
+
+    virtual void setAtlasFeature( QgsMapLayer *layer, const QgsFeature &feature ) = 0;
```

Comment:
Can't we get the Atlas layer from the layout? At best this argument is redundant, at worst it's going to cause confusion/issues.

Reply:
@nyalldawson good point, see last commit, I dropped the argument